src/components/virtual-gamepad-controls.js

```diff
@@ -0,0 +1,188 @@
+import nipplejs from "nipplejs";
+
+var THREE = AFRAME.THREE;
+var DEGREES = Math.PI / 180;
+var HALF_PI = Math.PI / 2;
```

Comment:
These constants are unused.

Reply:
Yeah. These can be deleted. I found that out after looking at the snap rotation component.